week2/readability/readability.c

```diff
@@ -0,0 +1,84 @@
+#include <cs50.h>
+#include <stdio.h>
+#include <ctype.h>
+#include <string.h>
+#include <math.h>
+
+int letters = 0;
+int words = 1;
+int sentences = 0;
+
+int count_letters(string text);
+int count_words(string text);
+int count_sentences(string text);
+string printLettersWordsSentences();
+
+int main(void)
+{
+    string text = get_string("Text: ");
+    int letters = count_letters(text);
+    int words = count_words(text);
+    int sentences = count_sentences(text);
+    printLettersWordsSentences(text);
+    float L = (float)letters / (float)words * 100;
```

Comment:
```suggestion
    int sentences = count_sentences(text);
    
    printLettersWordsSentences(text);
    
    float L = (float)letters / (float)words * 100;
```

Improves readability.